src/TransferRules.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  CHAIN_ID_BSC,
+  CHAIN_ID_ETH,
+  CHAIN_ID_TERRA,
+  CHAIN_ID_AURORA,
+  ChainId,
+  getForeignAssetTerra,
+  hexToUint8Array,
+  tryNativeToHexString,
+} from "@certusone/wormhole-sdk";
+import { PredicateArgs, Rule } from "./hooks/useTransferControl";
+import { terra } from "@certusone/wormhole-sdk";
+import { LCDClient } from "@terra-money/terra.js";
+import { getTerraConfig, getTokenBridgeAddressForChain } from "./utils/consts";
+
+const EthereumPandleAddress = "0X808507121B80C02388FAD14726482E061B8DA827";
+const BscPandleAddres = "0XB3ED0A426155B79B898849803E3B36552F7ED507";
+
+const lcd = new LCDClient(getTerraConfig(CHAIN_ID_TERRA));
+const getTerraClassic = (originChain: ChainId, originAssetHex: string) =>
+  getForeignAssetTerra(
+    getTokenBridgeAddressForChain(CHAIN_ID_TERRA),
+    lcd,
+    originChain,
+    hexToUint8Array(tryNativeToHexString(originAssetHex, originChain))
+  );
+
+(async () => {
```

Comment:
Should this function be here? Is this for debugging?

Reply:
you're right, good call, fixed